doc/integrations/suggestions.md

```diff
@@ -0,0 +1,9 @@
+#### Looking for ideas on making your own integrations?
```

Comment:
Didn't you create a page that explains how to make integrations for the hackathon?  Should we link that here as well?